src/components/Login/Login.js

```diff
@@ -140,6 +140,9 @@ const Login = ({ location, dependencies: { dopplerLegacyClient, sessionManager,
         });
       } else if (result.expectedError && result.expectedError.invalidLogin) {
         setErrors({ _error: 'validation_messages.error_invalid_login' });
+      } else if (result.expectedError && result.expectedError.wrongCaptcha) {
+        setErrors({ _error: 'validation_messages.error_invalid_captcha' });
+        console.log('invalid captcha', result);
```

Comment:
Is this also logged with _loggly_?

Reply:
Right now is logged in within loggly, this was an attempt to capture this as an expected error, but we need a message and an advise we could give to a user experimenting this issue. I'm not so sure about that...

Reply:
If I upload this code, It will stop showing in loggly and the user will see an special error for this scenario.

Reply:
Oh, it would be nice logging these errors with loggly also, right?

Reply:
Remember we would log only unexpected errors because of the log size? :trollface: Or do you suggest we do an exception just for this particular error? I'm not sure of merging this though...

Reply:
I was talking about logging the errors that could be making us troubles, without worry about if they are being shown to the user or not.

If you do not log this error, maybe you will have better information from the user when he calls support service, but you will not have any details about the request, response, etc.

Reply:
Ok, that seems nice!

Reply:
@andresmoschini fixed this issue.

Reply:
yo? yo no arreglé nada, a mi que me revisen...

Reply:
??? Sume loggear el error tambien, eso 😄